paddle/fluid/operators/detail/sendrecvop_utils.cc

```diff
@@ -63,6 +68,155 @@ void DeserializeFromMessage(const sendrecv::VariableMessage& msg,
   }
 }
 
+void SerializeToByteBuffer(const std::string& name, framework::Variable* var,
+                           const platform::DeviceContext& ctx,
+                           ::grpc::ByteBuffer* msg) {
+  sendrecv::VariableMessage request;
+  std::string header;
+  request.AppendToString(&header);
+  void* buf = malloc(1024);
+  void* payload;
+  size_t payload_size;
+  ProtoEncodeHelper e((char*)buf, 1024);
+  e.WriteString(1, name);
```

Comment:
Use kvarnameFieldNumber which defined in xxx.pb.h instead of number.
And the follows are the same problem.